README.md

```diff
@@ -1,20 +1,17 @@
 # kpt
 
-kpt is a git-native, schema-aware, extensible client-side tool for packaging, 
-customizing, validating, and applying Kubernetes resources.
+kpt is a package-centric toolchain that enables a WYSIWYG configuration 
+authoring, automation, and delivery experience, which simplifies managing
+Kubernetes platforms and KRM-driven infrastructure at scale by manipulating
+declarative Configuration as Data, separated from the code that transforms it.
```

Comment:
I really really like this description.

Reply:
I'm glad, because it appears in a lot of places. :-)